python/kiss_icp/pipeline.py

```diff
@@ -178,7 +178,7 @@ def _get_fps():
         avg_tra, avg_rot = sequence_error(self.gt_poses, self.poses)
         ate_rot, ate_trans = absolute_trajectory_error(self.gt_poses, self.poses)
         avg_fps = int(np.floor(_get_fps()))
-        avg_ms = 1e3 * (1 / avg_fps)
+        avg_ms = 1e3 * (1 / _get_fps())
```

Comment:
Wouldn't it be better to replace `np. floor` for `np.ceil` anyways? Having 0fps does not make sense! Thanks for this catch. This is what I would do:

```diff
diff --git a/python/kiss_icp/pipeline.py b/python/kiss_icp/pipeline.py
index e8d0b3a..21d8c66 100644
--- a/python/kiss_icp/pipeline.py
+++ b/python/kiss_icp/pipeline.py
@@ -177,8 +177,8 @@ class OdometryPipeline:
         # Run metrics evaluation
         avg_tra, avg_rot = sequence_error(self.gt_poses, self.poses)
         ate_rot, ate_trans = absolute_trajectory_error(self.gt_poses, self.poses)
-        avg_fps = int(np.floor(_get_fps()))
-        avg_ms = 1e3 * (1 / avg_fps)
+        avg_fps = int(np.ceil(_get_fps()))
+        avg_ms = int(np.ceil(1e3 * (1 / _get_fps())))
 
         self.results.append(desc="Average Translation Error", units="%", value=avg_tra)
         self.results.append(desc="Average Rotational Error", units="deg/m", value=avg_rot)
```